internal/reader/home_chain.go

```diff
@@ -39,6 +40,36 @@ type HomeChain interface {
 	services.Service
 }
 
+//go:generate mockery --name RMNConfigFetcher --output ./mocks/ --case underscore
+type RMNConfigFetcher interface {
+	GetRMNNodesInfo() ([]RMNNodeInfo, error)
+	GetRMNHomeConfigDigest() (cciptypes.Bytes, error)
+	// TODO: We could return the value for a selector, depending on how it's used
+	GetMinObservers() (map[cciptypes.ChainSelector]int, error)
+	GetMinSigners() (map[cciptypes.ChainSelector]int, error)
+}
+
```

Comment:
My concern is that probably this should not be part of `home_chain.go`. Since we are also querying other chains for `RMNRemote` config.

Reply:
In my mind they're totally separate flows, there is config that must be fetched from RMNHome and config from RMNRemote

Reply:
Anything that should be fetched from home can be part of the home chain reader. Anything not related to home chain would be part of the CCIP reader or we can create an RMN remote reader interface.

Reply:
I agree with this. My plan has shifted since this PR.
In short, we will use the home chain reader for the RMNHome and build another reader for RMNRemote. I'm not quite sure how to deal with the fact that not all the nodes can read RMNRemote but I'll deal with it in another PR.

Reply:
I also logically all the data coming from Home and Remote in different struct to avoid having to operate on the same struct by 2 different processes